tests/sessions_tests/test_backends.py

```diff
@@ -34,6 +34,8 @@
 from django.utils.encoding import force_text
```

Comment:
Not sure the renaming of this file is appropriate as part of this commit. There are test for the `SessionMiddleware` in this file too.